client.go

```diff
@@ -119,6 +123,34 @@ func NewClient(addr string, options ...func(c *Client) error) (*Client, error) {
 		return nil, c.scanErr()
 	}
 
+	// Initialize channels
+	c.notify = make(chan string)
```

Comment:
No buffering will cause processing stalls

Reply:
How big should the buffer size be? Should it be a fixed limit?